include/InlineEditing/inlineEditing.js

```diff
@@ -267,24 +267,37 @@ $(document).on('click', function (e) {
             var user_value = getInputValue(field, type);
 
             /**
```

Comment:
I would prefer to put comments/ detail info on fixes into the detail part of commit message, http://chris.beams.io/posts/git-commit/#seven-rules